sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionImplUtils.java

```diff
@@ -272,4 +277,44 @@ private static byte[] aesInternal(
       throw QueryExecutionErrors.aesCryptoError(e.getMessage());
     }
   }
+
+  public static ArrayData getSentences(
+      UTF8String str,
+      UTF8String language,
+      UTF8String country) {
+    if (str == null) return null;
+    Locale locale;
+    if (language == null && country == null) {
+      locale = Locale.US;
+    } else if (language == null) {
```

Comment:
This shouldn't happen, right? language is null but country is not null.

Reply:
Yes, this case shouldn't happen.

Reply:
This method (`def sentences(string: Column, country: Column): Column`) is not provided in the interface for end users.
<img width="503" alt="image" src="https://github.com/user-attachments/assets/ac2ead7f-5f00-4aaf-a082-eac918d19269">

Reply:
shall we add an assert to make it explicit?

Reply:
Updated.

Reply:
@cloud-fan 
This is my bad, it can `enter` this logic when the case is as follows:
```
 checkAnswer(
      df.selectExpr("sentences(str, null)"),
      Row(Seq(Seq("Hi", "there"), Seq("The", "price", "was"), Seq("But", "not", "now"))))
```
Here, `parameter be omitted` and `the value of parameter is null` are not the same concept.

Reply:
So, I removed this `assertion` and changed the coding, which seems to be more concise than before:
```
Locale locale;
    if (language != null && country != null) {
      locale = new Locale(language.toString(), country.toString());
    } else if (language != null) {
      locale = new Locale(language.toString());
    } else {
      locale = Locale.US;
    }
```